Marlin/src/lcd/dogm/dogm_font_data_ISO10646_1.h

```diff
@@ -26,19 +26,20 @@
   // reduced font (only sysmbols 1 - 127) - saves about 1278 bytes of FLASH
 
 /*
-  Fontname: -FontForge-ISO106461-Normal-R-Normal--11-80-96-96-C-60-WinRoman-0
-  Copyright:
+  Fontname: -Marlin6x12-Fixed-Medium-R-SemiCondensed--12-110-75-75-C-60-ISO10646-1
+  Copyright: Public domain terminal emulator font.  Share and enjoy. original font -Misc-Fixed-Medium-R-SemiCondensed--12-110-75-75-C-60-ISO10646-1
   Capital A Height: 7, '1' Height: 7
   Calculated Max Values w= 5 h=10 x= 5 y= 5 dx= 6 dy= 0 ascent= 8 len=10
-  Font Bounding box     w= 6 h=10 x= 0 y=-2
+  Font Bounding box     w= 6 h=12 x= 0 y=-2
   Calculated Min Values           x= 0 y=-2 dx= 0 dy= 0
-  Pure Font   ascent = 7 descent=-1
-  X Font      ascent = 7 descent=-1
+  Pure Font   ascent = 7 descent=-2
+  X Font      ascent = 8 descent=-2
   Max Font    ascent = 8 descent=-2
 */
-const u8g_fntpgm_uint8_t ISO10646_1_5x7[1302] U8G_FONT_SECTION(".progmem.ISO10646_1_5x7") = {
-  0x00,0x06,0x0a,0x00,0xfe,0x07,0x02,0x22,0x03,0xb1,0x01,0x7f,0xff,0x08,0xfe,0x07,
-  0xff,0x05,0x08,0x08,0x06,0x00,0x00,0x40,0xf0,0xc8,0x88,0x88,0x98,0x78,0x10,0x05,
+#include <clib/u8g.h>
```

Comment:
@p3p, @yhfudev — If `dogm_font_data_ISO10646_1.h` is included as (or from) C++ then is this #include in need of a change? I also notice that this include line appears two times in this file. Here, and then again on line 138. Maybe it only needs to be included once.

Reply:
we should only use `U8glib.h` I removed the need to bypass that header, it caused no end of issues

Reply:
It's already use U8glib.h after the merge with latest 2.0.x. see above

Reply:
on line 39 and 138 of this file it is still `<clib/u8g.h>`, that is what thinkyhead was referring to